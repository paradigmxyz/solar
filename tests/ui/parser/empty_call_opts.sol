function f() {
    f{gas:69}();
    f{}(); //~ ERROR: expected one of
}
