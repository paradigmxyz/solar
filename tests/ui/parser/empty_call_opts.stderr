error: expected one of `(`, `.`, `;`, `?`, or `[`, found `{`
  --> ROOT/tests/ui/parser/empty_call_opts.sol:LL:CC
   |
LL |     f{}();
   |      ^ expected one of `(`, `.`, `;`, `?`, or `[`
   |

error: aborting due to 1 previous error

